addons/binding/org.openhab.binding.zway/doc/GETTING_STARTED.md

```diff
@@ -0,0 +1,88 @@
+# Getting Started
+
+# Prepare Z-Way Server
+
+1. [Download](https://razberry.z-wave.me/z-way-server/) Z-Way v2.2.3 or newer (further information about installing Z-Way you can find [here](http://razberry.z-wave.me/index.php?id=24))
+2. Install openHAB Connector
+    - via "ZWave.Me App Store" use Token `sse_zwickau_beta`
+    - via Git
+
+    ```shell
+    cd /opt/z-way-server/automation/userModules
+    git clone https://github.com/pathec/ZWay-OpenHABConnector.git OpenHABConnector
+    service z-way-server restart
+    ```
+
+3. Instantiate the app once (under Configuration - Apps - Local Apps). Important: Change the selection from "Featured Apps" to "All Apps" to make the app visible. It is not necessary to make the configuration manually.
+
+# Prepare openHAB
+
+1. [Download](http://www.openhab.org/getting-started/downloads.html) openHAB 2 Beta Snapshot Offline or Online (further information about installing openHAB you can find [here](http://docs.openhab.org/installation/index.html))
```

Comment:
Better only mention to download openHAB 2 - we will soon have the release up there, no beta snapshot anymore!